### PR TITLE
add timeout config support and add more retry policy

### PR DESCRIPTION
**Pull request by tanhe123**
1. add timeout config support
2. add more retry strategies in order to use fun in very hard network environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ You can find more complicated examples here:
 - [函数计算](https://www.aliyun.com/product/fc)
 - [API Gateway](https://www.aliyun.com/product/apigateway)
 - [Fun 发布 2.0 新版本啦](https://yq.aliyun.com/articles/604490)
+- [三十分钟快速搭建serverless网盘服务](https://yq.aliyun.com/articles/613780)
```
Comment (vangie) [inline README.md]: serverless 前后加上空格

File: README-zh.md [hunk 2]
```diff
@@ -153,6 +153,7 @@ https://github.com/aliyun/fun/tree/master/examples
 - [函数计算](https://www.aliyun.com/product/fc)
 - [API Gateway](https://www.aliyun.com/product/apigateway)
 - [Fun 发布 2.0 新版本啦](https://yq.aliyun.com/articles/604490)
+- [三十分钟快速搭建serverless网盘服务](https://yq.aliyun.com/articles/613780)
```
Comment (vangie) [inline README-zh.md]: serverless 前后加上空格

File: lib/commands/config.js [hunk 3]
```diff
@@ -43,10 +43,23 @@ async function config() {
       choices: ['cn-beijing', 'cn-hangzhou', 'cn-shanghai', 'cn-shenzhen' , 'cn-hongkong',
         'ap-southeast-1', 'ap-southeast-2', 'ap-northeast-1', 'eu-central-1'],
       default: profile.defaultRegion
+    },
+    {
+      type: 'input',
+      name: 'timeout',
+      message: 'client timeout in seconds',
```
Comment (vangie) [inline lib/commands/config.js]: The timeout in seconds for each SDK client invoking

File: lib/commands/config.js [hunk 4]
```diff
@@ -70,7 +84,7 @@ async function config() {
       security_token: '',
       user_agent: 'fcli-0.1',
```
Comment (vangie) [inline lib/commands/config.js]: fcli-$pkg.version

File: README.md (hunk 1 above)
Comment (tanhe123) [reply README.md]: done

File: README-zh.md (hunk 2 above)
Comment (tanhe123) [reply README-zh.md]: done

File: lib/commands/config.js (hunk 3 above)
Comment (tanhe123) [reply lib/commands/config.js]: done

File: lib/commands/config.js (hunk 4 above)
Comment (tanhe123) [reply lib/commands/config.js]: removed

**Pull request merged**